package com.github.houbb.data.factory.core.exception;

import org.junit.jupiter.api.Test;

/**
* <p> DataFactoryRuntionException Tester. </p>
* <p> 2019-02-26 16:20:40.542 </p>
*
* @author binbin.hou
* @version 0.0.1-SNAPSHOT
*/
public class DataFactoryRuntionExceptionTest {

    /**
    *
    * Method: DataFactoryRuntionException()
    */
    @Test
    public void DataFactoryRuntionExceptionTest() throws Exception {
    }

    /**
    *
    * Method: DataFactoryRuntionException(message)
    */
    @Test
    public void DataFactoryRuntionExceptionMessageTest() throws Exception {
    }

    /**
    *
    * Method: DataFactoryRuntionException(message, cause)
    */
    @Test
    public void DataFactoryRuntionExceptionMessageCauseTest() throws Exception {
    }

    /**
    *
    * Method: DataFactoryRuntionException(cause)
    */
    @Test
    public void DataFactoryRuntionExceptionCauseTest() throws Exception {
    }

    /**
    *
    * Method: DataFactoryRuntionException(message, cause, enableSuppression, writableStackTrace)
    */
    @Test
    public void DataFactoryRuntionExceptionMessageCauseEnableSuppressionWritableStackTraceTest() throws Exception {
    }


}
